blueoil/datasets/pascalvoc_base.py

```diff
@@ -184,16 +182,16 @@ def _all_image_ids(self, data_type=None, is_debug=False):
             raise ValueError("Must provide data_type = train or val or trainval or test")
 
         filename = os.path.join(self.imagesets_dir, data_type + ".txt")
-        df = pd.read_csv(
-            filename,
-            delim_whitespace=True,
-            header=None,
-            names=['image_id'])
+        image_id = list()
+
+        with open(filename) as f:
```

Comment:
Why don't you use csv module?

Reply:
The lines here are about reading a txt file, not csv file.